internal_packages/ui-less-is-more/README.md

```diff
@@ -0,0 +1,28 @@
+# Less is More
+A minimal approach to email using [Nylas N1](https://nylas.com/n1).
+
+[![Message](./images/composer-window.png)](./images/composer-window.png)
+
+[More images can be found at →](http://www.alexanderadkins.com/projects/less-is-more)
+
+## Installation
+
+[Download Less Is More](https://github.com/P0WW0W/less-is-more/archive/master.zip)
+
+##### Step 1
```

Comment:
Can you drop the installation steps and screenshot, since folks can use N1's theme picker to try it out / select it?